language/Pattern-Matching.md

```diff
@@ -23,16 +23,17 @@ Patterns can also be used when introducing functions. For example:
 example x y z = x * y + z
 ```
 
-The following pattern types are supported:
+The following forms can be used for matching:
```

Comment:
Yes, what he said :). I decided on that wording to draw the distinction we discussed between guards, which are conditional in nature, and what I would call forms of patterns, which match against an argument's shape or value.